docs/updating-command-reference.md

```diff
@@ -0,0 +1,39 @@
+### Updating the kubeadm command reference documentation
+
+kubeadm uses [Cobra](https://github.com/spf13/cobra) as a CLI library and the command line reference
+documentation is generated automatically. The generated output can be found here:
+* https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm/
+
+If a command or a command flag is added or removed this has to be reflected in the documentation on a new release.
+The `kubernetes/website` repository has the tooling to facilitate this, but some manual steps are still required.
+
+#### Example scenario: adding a new sub-command
+
+- Implemented a new kubeadm configuration sub-command called `kubeadm config newcommand`.
+- Run `./hack/update-generated-docs.sh`. This will generate new files in the folder `./docs/admin/` called `*newcommand*`.
+- If you are sending a PR to `kubernetes/kubernetes` make sure that you add these files
```

Comment:
Before adding files to k/k you should run `hack/update-generated-docs.sh`. it replace cobra helps with placeholders